app/src/main/java/com/findmyrecycling/ui/theme/Color.kt

```diff
@@ -6,4 +6,5 @@ val Purple200 = Color(0xFFBB86FC)
 val Purple500 = Color(0xFF6200EE)
 val Purple700 = Color(0xFF3700B3)
 val Teal200 = Color(0xFF03DAC5)
-val RecyclingGreen = Color(0xFF009900)
\ No newline at end of file
+val RecyclingGreen = Color(0xFF009900)
+val RecyclingGray = Color(0xFFcccbc8)
```

Comment:
I noticed that the preview screen shot in the readMe file used gray background for your text fields so I added a Gray color to the theme.  0xFFcccbc8

Reply:
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.